### DISCUSSION TITLE
Issue with temperature (float) values as extra_features

### DISCUSSION THREAD

**Discussion by fsGORM** [google/lightweight_mmm#243] [General]
Hi 

I'm trying to add temperature averages as extra_features in my MMM model, however when I fit the model it returns an error that says 'tuple index out of range'. My extra_features set contains two dataframe columns (with float point values) and I've saved my extra_features variable to be a list containing said columns, but when I try to fit a model with the scaled version of extra_features, it tells me the tuple index is out of range. 

Anyone know a fix to this?

**Comment by statm3n** [comment]
extra_features should be in jnp.ndarray for fit. Maybe check if your extra_features are in the right format? 

https://github.com/google/lightweight_mmm/blob/8152c43c8b52eb8c020c1c54c7e4bfa5140e42d0/lightweight_mmm/utils.py#L195 can help convert pandas dataframe to jax arrays.

**Comment by fsGORM** [comment]
I've tried doing that with
`extra_features_col = jnp.array(data["gns_temp"].values) `
LightweightMMM expects `extra_features_plates_shape = (extra_features.shape[1],)`
Which is where it throws the error. 
The shape of my array: (103,)
